test/integration/src/main/java/io/pravega/test/integration/compatibility/CompatibilityChecker.java

```diff
@@ -221,7 +232,99 @@ private void checkStreamDelete() {
         assertFalse(streamManager.checkStreamExists(scopeName, streamName));
     }
 
-    public static void main(String[] args) throws DeleteScopeFailedException {
+    /**
+     * This method attempts to validate abort functionality after creating a transaction.
+     * @throws TxnFailedException if unable to commit or write to the transaction.
+     */
+    @Test(timeout = 20000)
+    private void checkTransactionAbort() throws TxnFailedException {
+        String scopeName = "transaction-abort-test-scope";
+        String streamName = "transaction-abort-test-stream";
+        streamManager.createScope(scopeName);
```

Comment:
This part of code is repeated at various places in this file. Can you create a method extracting these line. The name of the method could be like `createScopeAndStream(String scopeName, String streamName, StreamConfig streamConfig)`

Reply:
added a new method createScopeAndStream(String scopeName, String streamName).